include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -40,23 +40,46 @@ namespace NovelRT::Maths
         float x;
         float y;
 
+        /**
+         * @brief Construct new GeoVector2F with X and Y components set to 0
+         * @returns Newly constructed GeoVector2F
+         */
         GeoVector2F() noexcept : x(0.0f), y(0.0f)
         {
         }
+
+        /**
+         * @brief Construct new GeoVector2F with X and Y components with x and y paramters
+         * @param x Set X Component Value
+         * @param y Set Y Component Value
+         * @returns Newly constructed GeoVector2F
+         */
         GeoVector2F(float x, float y) noexcept : x(x), y(y)
         {
         }
 
+        /**
+         * @brief Compute a normalized vector of this vector
+         * @returns A Unit Vector in GeoVector2F that have the length of 1
+         */
```

Comment:
Changed the US english to UK english. Grammar and punctuation fixes.

```suggestion
        /**
         * @brief Computes a normalised GeoVector2F of this GeoVector2F.
         * @returns A GeoVector2F that has the length (or magnitude) of 1.
         */
```